arxivdigest/api/database.py

```diff
@@ -252,6 +252,29 @@ def get_topic_recommendations(ids):
 
         return users
 
+def get_user_topics(ids):
```

Comment:
should be merged with get_user_feedback_topics such that the same topics that are blocked are sent to the systems, such that they can know upfront which topics are blocked.

This function should probably not exist.

Reply:
problem is the user feedback only returns topics that have been interleaved, where this function also needs to return the topics that have been recommended but not yet been interleaved